content/switching-to-v0-mangling-on-nightly.md

```diff
@@ -0,0 +1,190 @@
++++
+path = "2025/10/28/switching-to-v0-mangling-on-nightly"
+title = "Switching to Rust's own mangling scheme on nightly"
+authors = ["David Wood"]
+
+[extra]
+team = "the compiler team"
+team_url = "https://www.rust-lang.org/governance/teams/compiler"
++++
+
+**TL;DR:** rustc will use its own "v0" mangling scheme by default on nightly
+versions instead of the previous default scheme which re-used C++'s mangling
```

Comment:
I found this long sentence hard to read, without any punctuation, and with repeated "scheme".

Is this more readable?
```suggestion
versions instead of the previous default, which re-used C++'s mangling
```

Reply:
Fixed